capstone/feed-fetching/get_vt_feed.py

```diff
@@ -0,0 +1,51 @@
+import bz2
+import io
+import itertools
+import json
+import os
+import requests
+from datetime import datetime, timedelta
+from elasticsearch import Elasticsearch
+
+es = Elasticsearch('elastic-stack-instance:9200')
+apikey = os.environ.get('VTKEY')  # is there a better way to store api key?
```

Comment:
Global variables must be capital letters
```suggestion
API_KEY = os.environ.get('VTKEY')  # is there a better way to store api key?
```